example/busyring/parameters.hpp

```diff
@@ -33,13 +33,12 @@ struct ring_params {
     ring_params() = default;
 
     std::string name = "default";
-    unsigned num_cells = 10;
-    unsigned ring_size = 10;
-    double min_delay = 10;
-    double duration = 100;
+    unsigned num_cells = 5000;
```

Comment:
How long would you expect this to run on a reasnably modern desktop/laptop? 
An ideal runtime for people trying would -- for me -- be around 10s.

Reply:
I want something that does an interesting amount of computation, and this is a pretty good sweet spot that won't take too long on the CI/CD

Reply:
If it doesn't bog down CI too much I am good